subsys/bluetooth/controller/ll_sw/nordic/lll/lll_central_iso.c

```diff
@@ -1237,3 +1133,147 @@ static void isr_done(void *param)
 
 	lll_isr_cleanup(param);
 }
+
+static void payload_count_flush(struct lll_conn_iso_stream *cis_lll)
+{
+	if (cis_lll->tx.bn) {
+		uint64_t payload_count;
+		uint8_t u;
+
+		payload_count = cis_lll->tx.payload_count + cis_lll->tx.bn_curr - 1U;
+		u = cis_lll->nse - ((cis_lll->nse / cis_lll->tx.bn) *
+				    (cis_lll->tx.bn - 1U -
+				     (payload_count % cis_lll->tx.bn)));
+		if (((cis_lll->tx.payload_count / cis_lll->tx.bn) + cis_lll->tx.bn_curr +
+		     cis_lll->tx.ft + u) <
+		    (cis_lll->event_count + se_curr + 2U)) {
```

Comment:
Here the comparison between `cis_lll->rx.payload_count / cis_lll->rx.bn` and `event_count` is significantly different. 

Can you elaborate on why?